### PR TITLE
Add Windows support

### PR DESCRIPTION
**Pull request by JoelBCarter**
Add cross-platform support for all NPM scripts.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,12 +25,12 @@
     "start": "nodemon --ext .ts --watch ./src --exec \"node --max-old-space-size=8192 --experimental-specifier-resolution=node --loader ts-node/esm ./src/index.ts\"",
     "launch": "node --max-old-space-size=8192 --experimental-specifier-resolution=node ./dist/node-esm/index.js",
     "build": "yarn lint && yarn compile",
-    "clean": "rm -f tsconfig.tsbuildinfo && rm -r -f ./build && rm -r -f ./.serverless && rm -r -f ./.webpack",
-    "compile": "rm -r -f ./dist && NODE_OPTIONS=--max-old-space-size=8192 yarn compile-rollup",
```
Comment (JoelBCarter) [inline package.json:29]: @arietrouw double-check me on this one but I don't think we need V8 memory options for compilation.  We should revisit hardcoding it too as it's really dictated by the architecture it's running on and we should make it configurable.

**Comment by JoelBCarter** [comment]
I checked on both Linux & Windows and it build/runs.

File: package.json [hunk 2]
```diff
@@ -25,12 +25,12 @@
...
+    "clean": "rimraf tsconfig.tsbuildinfo && rimraf ./build && rimraf ./.serverless && rimraf ./.webpack",
+    "compile": "rimraf ./dist && yarn compile-rollup",
     "compile-rollup": "rollup -c rollup-config/index.js",
     "compile-tsc": "tsc -p tsconfig.build.json",
-    "reinstall": "rm -r -f ./node_modules && rm -f ./yarn.lock && rm -f ./yarn-error.log && yarn install && yarn outdated",
-    "license": "yarn license-checker --exclude 'MIT, ISC, Apache-2.0, BSD, BSD-2-Clause, CC-BY-4.0, Unlicense, CC-BY-3.0, CC0-1.0'"
+    "reinstall": "rimraf ./node_modules && rimraf ./yarn.lock && rimraf ./yarn-error.log && yarn install && yarn outdated",
```
[the remark this answers is not in the document]
Comment (JoelBCarter) [reply package.json:32]: Good catch...the Terminator principle of delete yourself last
image

[the remark this answers is not in the document]
Comment (jonesmac) [reply package.json]: If we build an XYO dev principles...this definitely made the list!

**Review by jonesmac — APPROVED** [review]
Works on my (non-windows) machine!

**Comment by JoelBCarter** [comment]
@jonesmac I fixed it by not removing the yarn lockfile each time we run `yarn reinstall`.  Yarn's precedence is to:

1. Infer/use whatever EOL the existing file has
2. If there's no lockfile, use the OS default line endings (different for Mac/Windows)

So by not blowing it away it'll preserve the existing Linux line endings which Windows is just fine with.  Since yarn is deterministic we didn't need to be blowing away the file each time anyway.

**Pull request merged**